includes/documents/class-wcpdf-invoice.php

```diff
@@ -457,7 +457,7 @@ public function get_pdf_settings_fields( $option_name ) {
 				'args'			=> array(
 					'option_name'	=> $option_name,
 					'id'			=> 'invoice_number_search',
-					'description'   => __( 'Can potentially slow down the search process.', 'woocommerce-pdf-invoices-packing-slips' ),
+					'description'   => __( 'Can potentially slow down the search process. You can also search orders by invoice numbers using search type selector, which is faster than this option.', 'woocommerce-pdf-invoices-packing-slips' ),
```

Comment:
Maybe you should mention something about the HPOS since the customer might not be aware that this is a feature of HPOS.

Reply:
@MohamadNateqi I agree, I have changed the code!